src/app/sections/ProbeDetails.svelte

```diff
@@ -194,6 +199,13 @@ h2 {
             </div>
         </div>
         {/if}
+        <div class="drawer-section">
+            <div class="helper-text--01">
+                <a href={getProbeDictURL($store.probe.apiName)} target="_blank">
+                    open in Probe Dictionary <ExternalLink size=12 />
```

Comment:
```suggestion
                    more info <ExternalLink size=12 />
```

Reply:
Not sure if this is the right copy, but worth considering what the right copy is here for the end-user